crates/rolldown/tests/esbuild/default/forbid_const_assign_when_bundling/test.config.json

```diff
@@ -6,5 +6,6 @@
         "import": "entry.js"
       }
     ]
-  }
-}
\ No newline at end of file
+  },
+  "expectExecuted": false
```

Comment:
Is this error catchable? If it is, you could do


```
let err;

try { ... } catch (e) { err = e }

assert(typeof err !== 'undefined`)
```

Reply:
test_script cannot catch re-assignment errors (`TypeError: Assignment to constant variable.`), so I've changed it to catch them in entry.js.
https://github.com/rolldown/rolldown/pull/538/commits/4e50ce2d03426700aed0e575e425afa08fa418ef